envs/dev/templates/team-geordi.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: team-geordi
+  namespace: argocd
+  finalizers:
+  - resources-finalizer.argocd.argoproj.io
+spec:
+  project: default
+  destination:
+    namespace: ho11y
```

Comment:
namespace here is argocd - that is where apps will be created

Reply:
Understood, fixed this.